### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by odespard**
I added a paragraph to the Installation section to explain an alternative means of downloading mono which circumvented an error which arose when installing it using homebrew.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,9 +63,20 @@ requires you to add the ppa to your trusted sources!
...
+4.  Link homebrew mono to pseudo mono folder: `sudo ln -s /opt/homebrew/Cellar/mono/6.12.0.182 /Library/Frameworks/Mono.framework/Versions/Current`. Here, `6.12.0.182` is the brew-installed mono version, please check your installed version. Navigate to `/Library/Frameworks/Mono.framework/Versions` and run `ls -l` to verify that the link `Current` points to `/opt/homebrew/Cellar/mono/6.12.0.182`. If `Current` points to a different installation and/or `/opt/homebrew/Cellar/mono/6.12.0.182` is referenced by a different link, delete the corresponding links and run `sudo ln -s /opt/homebrew/Cellar/mono/6.12.0.182 Current`.
 5.  Install pythonnet: `pip install pythonnet`.
 
+NOTE, Homebrew installs the most recent version of mono, which may give rise to the following error on the M1 platform (which is due to an incompatible architecture for the files in the mono library): 
+RuntimeError: Failed to create a default .NET runtime, which would
```
Comment (mschwoer) [inline README.md:70]: (nit) maybe format the error message as code to make it more redable

**Review by mschwoer — APPROVED** [review]
congrats to your first PR :-)
maybe this fixes also the issue I had with mono in Docker on M3 ..

**Comment by jalew188** [comment]
Thank you @odespard for this fix. I think the testing failure is because the `main` branch is out-of-date in testing settings, please checkout the `development` branch and then add the fix there.

**Comment by odespard** [comment]
OK. I have forked from the Development branch and opened a new pull request. I hope that works.

**Pull request closed without merging**